contents/docs/integrate/server/ruby.mdx

```diff
@@ -170,25 +174,101 @@ posthog.capture({
 })
 ```
 
+
 ### Feature flags
 
-> Note that to use feature flags you must specify `personal_api_key` when initializing PostHog.
+PostHog's [Feature Flags](/docs/user-guides/feature-flags) allow you to safely deploy and roll back new features.
+
+When using them with one of libraries, you should check if a feature flag is enabled and use the result to toggle functionality on and off in you application.
+
+**How to check if a flag is enabled**
+
+> **Note:** Whenever we face an error computing the flag, the library returns `Nil`, instead of `true` or `false`.
+
+
+```c
+posthog.is_feature_enabled('beta-feature', 'distinct id')
+# returns True or False or Nil
+```
+
+**Example use case**
+
+Here's how you might send different users a different version of your homepage, for example:
+
+```c
+def homepage(request):
+    template = "new.html" if posthog.is_feature_enabled('new_ui', 'distinct id') else "old.html"
+    return render_template(template, request=request)
+```
+
+> **Note:** Feature flags are persistent for users across sessions. Read more about feature flag persistence on our [dedicated page](/docs/user-guides/feature-flags#feature-flag-persistence).
+
+**Get a flag value**
+
+If you're using multivariate feature flags, you can also get not just whether the flag is enabled, but what value its enabled to.
 
-#### Checking if a feature is enabled
+> **Note:** Whenever we face an error computing the flag, the library returns `Nil`, instead of `true` or `false` or a string variant value.
```

Comment:
```suggestion
> **Note:** Whenever we face an error computing the flag, the library returns `Nil`, instead of `true`, `false`, or a string variant value.
```

@neilkakkar Other places in the docs we say we return `None`, but here we say we return `Nil` instead. Does this make a difference, or is one incorrect?

Reply:
In `ruby`, the null value is `Nil`. In `python`, it's `None`, and in javascript, it's `undefined`, hence the 3 in the docs for the 3 different libraries :)

Reply:
I love learning about code! Thanks :)